Java/PoolAndResourceFile/README.md

```diff
@@ -0,0 +1,44 @@
+---
+services: Batch
+platforms: java
+author: xingwu
```

Comment:
azure-samples probably actually parses this data -- since we don't live in their "world" is there any value in having this?